crates/bdk/src/wallet/tx_builder.rs

```diff
@@ -184,12 +184,18 @@ impl<'a, D, Cs: Clone, Ctx> Clone for TxBuilder<'a, D, Cs, Ctx> {
 // methods supported by both contexts, for any CoinSelectionAlgorithm
 impl<'a, D, Cs: CoinSelectionAlgorithm, Ctx: TxBuilderContext> TxBuilder<'a, D, Cs, Ctx> {
     /// Set a custom fee rate
+    /// The fee_rate method sets the fee rate using a FeeRate type in either satoshis/vbyte (sats/vbyte)
```

Comment:
I needed to use this method and came across your PR. I think it is useful to comment a little more how to set fee rates since it is always a source of confusion even for people familiar with Bitcoin, let alone newcomers.
That being said I'd be a little more verbose here, and write something like this:
```
/// Set the mining fee paid by the transaction as a rate on its size.
    /// This means that the total fee paid is equal to this rate * size of the transaction in virtual Bytes (vB) or Weigth Unit (wu).
    /// This rate is internally expressed in satoshis-per-virtual bytes (sats/vB), but can also be set by:
    /// * sats/kvB (1000 sats/kvB == 1 sats/vB) using FeeRate::from_sat_per_kvb
    /// * btc/kvB (0.00001000 btc/kvB == 1 sats/vB) using FeeRate::from_btc_per_kvb
    /// * sats/kwu (250 sats/kwu == 1 sats/vB) using FeeRate::from_sat_per_kwu
    /// Default is 1 sat/vB (see min_relay_fee)
```

But maybe that's *too* verbose now so just a suggestion

Reply:
I think these extra details are a good idea. I've updated the PR to include your suggestion.